### ISSUE TITLE
Missing separator with XKCD style passwords

### ISSUE THREAD

**Issue by jamesponddotco** [Luzifer/password#6]
Would it be possible to add a separator to each word with the XKCD style password? Something like what is done with [xkcdpass](https://github.com/redacted/XKCD-password-generator) or [1Password's strong password generator](https://1password.com/password-generator/).

By default it could be `correct-horse-battery-staple`, but offering a setting to choose the separator would be even better =)

**Comment by jamesponddotco** [comment]
Seems to be working perfectly, thanks, @Luzifer!

```
james@melanie.jamespond.local:~$ curl -s "https://passwd.fyi/v1/getPassword?date=false&length=10&xkcd=true&separator=-"
Sidecar-Bastion-Mortician-Railroad-Freezer-Blackbird-Carillon-Inferno-Nightgown-Stereo
```

**Comment by Luzifer** [comment]
I just published the release `v2.2.0` in which the CLI as well as the server version support the new separator parameter.

And as you already noticed, my hosted version is also up to date. :slightly_smiling_face:

Thank you for the suggestion! :slightly_smiling_face:
